src/eastwood/lint.clj

```diff
@@ -1010,9 +1010,8 @@ Eastwood has other forms of effective, safe parallelism now. Falling back to seq
        -main))
 
   ([& opts]
-   (if (and
-        (= 1 (count opts))
-        (string? (first opts)))
-     (eastwood-from-cmdline (edn/read-string (first opts)))
-     (let [parsed (->> opts (interpose " ") (apply str) edn/read-string)]
-       (eastwood-from-cmdline parsed)))))
+   (let [reducer #(merge %1 (if (string? %2)
```

Comment:
I'd like this codebase to keep following https://stuartsierra.com/2019/09/11/clojure-donts-short-fn-syntax